astropy/table/table_helpers.py

```diff
@@ -96,9 +96,9 @@ def simple_table(size=3, cols=None, kinds='ifS', masked=False):
     letters = np.array([c for c in string.ascii_letters])
     for jj, kind in zip(range(cols), cycle(kinds)):
         if kind == 'i':
-            data = np.arange(1, size + 1, dtype=int) + jj
+            data = np.arange(1, size + 1, dtype=np.int64) + jj
```

Comment:
I was mostly worried about Windows where int32 is the default, so I'm not sure what `dtype=int` does, but I'm guessing it's int32.